lib/hammer_cli/settings.rb

```diff
@@ -59,9 +64,16 @@ def self.path_history
       @path_history
     end
 
+    def self.default_values
```

Comment:
How about renaming it to `default_settings` ?
It's probably just because of this PR's context, but the name associated me different functionality.